drivers/eks/eks_driver.go

```diff
@@ -277,6 +286,10 @@ func (state *state) validate() error {
 		return fmt.Errorf("maximum nodes cannot be less than minimum nodes")
 	}
 
+	if state.NodeVolumeSize < 1 {
```

Comment:
Won't clusters that already exist have a `NodeVolumeSize` of 0?  Will this cause existing clusters to get stuck in error loops?  Seems like we also need some logic to default this to something unless I'm mistaken.

Reply:
I kind of inferred this part. EKS automatically configured the volume size to be 20, so I can't imagine one being 0 unless it's configured that way. If we want to allow it I can just check that it non-negative.